### ISSUE TITLE
Implement Nice squash feature

### ISSUE THREAD

**Issue by lakruzz** [Praqma/git-phlow#49]
The developer needs to have easy access to clean up local history before deliver

**Comment by groenborg** [comment]
I agree but I think we good to have a meeting about this where we discuss how to do it
